app/policies/membership_policy.rb

```diff
@@ -3,6 +3,10 @@ def destroy?
     user.super_admin? || organisation_admin? || (group_admin? && record.editor?)
   end
 
+  def update?
```

Comment:
we are using update as as the policy because it's a standard route and makes sense given the current requirements.

It could be that we want something more specific in the future, e.g `can_change_role?` but for now I think this is the simplest option?